CHANGES.md

```diff
@@ -15,6 +15,11 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
+[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+
+[1.41.0](https://github.com/bird-house/birdhouse-deploy/tree/1.41.0) (2023-11-30)
+------------------------------------------------------------------------------------------------------------------
+
```

Comment:
Empty CHANGES.md ?!

Reply:
I think there was an update and version bbump that happened at the same time of my change and the merge went wrongly. I've reverted my change and re bumped  version. It should be fixed. Can you confirm ?

Reply:
Yes, this time the diff looks good.

Reply:
okay I will merge once the tests finish running